src/drivers/net/ena.c

```diff
@@ -206,23 +207,39 @@ static int ena_create_admin ( struct ena_nic *ena ) {
 	}
 	memset ( ena->aq.req, 0, aq_len );
 
+	/* Allocate AENQ queue */
+	ena->aenq.events = malloc_phys ( aenq_len, aenq_len );
+	if ( ! ena->aenq.events ) {
+		rc = -ENOMEM;
+		goto err_alloc_aenq;
+	}
+	memset ( ena->aenq.events, 0, aenq_len );
+
 	/* Program queue addresses and capabilities */
 	ena_set_base ( ena, ENA_ACQ_BASE, ena->acq.rsp );
 	ena_set_caps ( ena, ENA_ACQ_CAPS, ENA_ACQ_COUNT,
 		       sizeof ( ena->acq.rsp[0] ) );
 	ena_set_base ( ena, ENA_AQ_BASE, ena->aq.req );
 	ena_set_caps ( ena, ENA_AQ_CAPS, ENA_AQ_COUNT,
 		       sizeof ( ena->aq.req[0] ) );
+	ena_set_base ( ena, ENA_AENQ_BASE, ena->aenq.events );
+	ena_set_caps ( ena, ENA_AENQ_CAPS, ENA_AENQ_COUNT,
+		       sizeof ( ena->aenq.events[0] ) );
 
-	DBGC ( ena, "ENA %p AQ [%08lx,%08lx) ACQ [%08lx,%08lx)\n",
+	DBGC ( ena, "ENA %p AQ [%08lx,%08lx) ACQ [%08lx,%08lx) AENQ [%08lx,%08lx)\n",
```

Comment:
Please wrap to 80 characters (reflowing onto the next line immediately before `ena, `).  Please also check that the printed debug message will (with typical values) fit within 80 characters and, if not, split to two debug messages.